src/plugins/notebook/components/Notebook.vue

```diff
@@ -133,6 +134,11 @@
                     To start a new entry, click here or drag and drop any object
                 </span>
             </div>
+            <progress-bar
+                v-if="savingTransaction"
+                class="c-telemetry-table__progress-bar"
+                :model="{ progressPerc: undefined }"
```

Comment:
nitpick, but kinda feel like this should be `null` as it is an initialized value.

that would require a change to `ProgressBar` and anything else that calls up an indefinite progress bar so I'll leave that up to you whether or not you think it's worth it. In general I'm trying to not use `null` and `undefined` interchangeably as there is some convention for the difference between the two:

https://stackoverflow.com/a/5076962/6596223

Reply:
this comment is non-blocking for this PR

Reply:
Agreed, as this is related to another component, I've created an issue to address this: https://github.com/nasa/openmct/issues/6085
I don't want to push this out any further or add any unrelated changes to this PR, `ProgressBar` is used in multiple other places. Good point though!

Reply:
This is great, and a good first issue for open source contributors. Thanks for creating that!